backend/tracim_backend/tests/functional/test_workspaces.py

```diff
@@ -5275,10 +5275,7 @@ def test_api__get_content_path__ok__200__nominal_case(
         web_testapp.authorization = ("Basic", ("test@test.test", "test@test.test"))
 
         my_file_path = web_testapp.get(
-            "/api/workspaces/{}/contents/{}/path".format(
-                workspace.workspace_id, my_file.content_id
-            ),
-            status=200,
+            "/api/contents/{}/path".format(my_file.content_id), status=200,
```

Comment:
Can you explain the behavior if content is not accessible to the connected user ?
A test may be nice.

Reply:
Good point, I added a test (it returns 400 Content not found in database) @inkhey